package.json

```diff
@@ -14,6 +14,16 @@
     "import": "./dist/index.js",
     "require": "./dist/index.cjs"
   },
+  "tags": [
+    "url",
+    "urlpattern",
+    "url-pattern",
+    "type-script",
+    "javascript",
+    "browser",
+    "node",
+    "polyfill"
```

Comment:
Maybe "w3c", "wicg", "web" as well?

Is there anywhere we can see common tags?